filebeat/module/apache2/access/_meta/fields.yml

```diff
@@ -0,0 +1,107 @@
+- name: access
+  type: group
+  description: >
+    Contains fields for the Apache2 HTTPD access logs.
+  fields:
+    - name: remote_ip
+      type: keyword
+      description: >
+        Client IP address.
+    - name: user_name
+      type: keyword
+      description: >
+        The user name used when basic authentication is used.
+    - name: method
+      type: keyword
+      example: GET
+      description: >
+        The request HTTP method.
+    - name: url
+      type: keyword
+      description: >
+        The request HTTP URL.
+    - name: http_version
+      type: keyword
+      description: >
+        The HTTP version.
+    - name: response_code
+      type: long
+      description: >
+        The HTTP response code.
+    - name: body_sent.bytes
+      type: long
+      format: bytes
+      description: >
+        The number of bytes of the server response body.
+    - name: referrer
+      type: keyword
+      description: >
+        The HTTP referrer.
+    - name: agent
+      type: text
+      description: >
+        Contains the un-parsed user agent string. Only present if the user
+        agent Elasticsearch plugin is not available or not used.
+    - name: user_agent
+      type: group
+      description: >
+        Contains the parsed User agent field. Only present if the user
+        agent Elasticsearch plugin is available and used.
+      fields:
+        - name: device
+          type: keyword
+          description: >
+            The name of the physical device.
+        - name: major
+          type: long
+          description: >
+            The major version of the user agent.
+        - name: minor
+          type: long
+          description: >
+            The minor version of the user agent.
+        - name: patch
+          type: long
+          description: >
+            The patch version of the user agent.
+        - name: name
+          type: keyword
+          example: Chrome
+          description: >
+            The name of the user agent.
+        - name: os
```

Comment:
These are created by the user agent processor, which is not that flexible when it comes to field names: https://www.elastic.co/guide/en/elasticsearch/plugins/master/using-ingest-user-agent.html

We could maybe do renamings, but I'd say the complexity is not worth it.